src/mono/sample/wasm/browser-threads-minimal/Program.cs

```diff
@@ -93,6 +141,29 @@ public static async Task<string> FetchBackground(string url)
             return r;
         }
 
+        [ThreadStatic]
+        public static int meaning = 42;
+
+        [JSExport]
+        public static async Task TestTLS()
+        {
+            Console.WriteLine($"smoke {meaning}: TestTLS 1 ManagedThreadId:{Thread.CurrentThread.ManagedThreadId}, SynchronizationContext: {SynchronizationContext.Current?.GetType().FullName ?? "null"}");
```

Comment:
This file is pretty magical, so without comments I'm not sure anyone else will get value out of it. I don't think that's necessarily a problem though

Reply:
This sample is messy work in progress. We will have to create unit tests out of this at some point.